tools/universe/package.py

```diff
@@ -0,0 +1,34 @@
+from .package_manager import PackageManager
+from .version_resolver import VersionResolver
+
+
+class Package:
+
+    def __init__(self, name, version, version_resolver=None):
+        self._name = name
+        self._version = version
+
+        self._set_version_resolver(version_resolver)
+
+    def _set_version_resolver(self, version_resolver):
```

Comment:
Python likes double newlines between functions (here and below)

Reply:
As far as I am aware, PEP8 only requires the double newlines between module-level functions or classes, for class methods, or other whitespace, a single blank line is required. I fill run `flake8` on the files here again to be sure.